packages/gatsby-plugin-google-analytics/src/gatsby-ssr.js

```diff
@@ -18,31 +18,43 @@ exports.onRenderBody = (
   { setHeadComponents, setPostBodyComponents },
   pluginOptions
 ) => {
-  if (process.env.NODE_ENV === `production`) {
-    let excludeGAPaths = []
-    if (typeof pluginOptions.exclude !== `undefined`) {
-      const Minimatch = require(`minimatch`).Minimatch
-      pluginOptions.exclude.map(exclude => {
-        const mm = new Minimatch(exclude)
-        excludeGAPaths.push(mm.makeRe())
-      })
-    }
+  if (process.env.NODE_ENV !== `production`) {
+    return null
+  }
 
-    const gaCreateOptions = {}
-    for (const option in knownOptions) {
-      if (typeof pluginOptions[option] === knownOptions[option]) {
-        gaCreateOptions[option] = pluginOptions[option]
-      }
+  // Lighthouse recommends pre-connecting to google analytics
+  setHeadComponents([
+    <link
+      rel="dns-prefetch preconnect"
```

Comment:
```suggestion
      rel="preconnect dns-prefetch"
```
Put `preconnect` first otherwise, the browser will always use `dns-prefetch` Order matters.

Reply:
Thanks for noticing this!

Reply:
oh yeah sorry my bad 😄 @yogeshkotadiya thanks for having another pair of eyes